README.md

```diff
@@ -1,173 +1,134 @@
-# Cloudinary Vue SDK
+Cloudinary Vue SDK
+=========================
+[![Build Status](https://api.travis-ci.com/cloudinary/cloudinary-vue.svg?branch=master)](https://app.travis-ci.com/github/cloudinary/cloudinary-vue)
+## About
+The Cloudinary Vue SDK allows you to quickly and easily integrate your application with Cloudinary.
+Effortlessly optimize and transform your cloud's assets.
 
-[![Build Status](https://travis-ci.org/cloudinary/cloudinary-vue.svg?branch=master)](https://travis-ci.org/cloudinary/cloudinary-vue)
 
-Cloudinary is a [cloud-based service](https://cloudinary.com/solutions) that provides an end-to-end image and video management solution
-including uploads,storage, manipulations, optimizations and delivery. 
-All your media resources are optimized and delivered through a fast CDN using industry best practices.
+### Additional documentation
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [Vue SDK Guide](https://cloudinary.com/documentation/vue_integration).
 
-Using Cloudinary's Vue.js SDK, you can easily perform smart image and video manipulations 
-using code that integrates seamlessly with your existing Vue.js application, and deliver optimized and responsive media to your users.
 
-This Readme provides basic installation and usage information. For the complete documentation, see the [Vue.js SDK guide](https://cloudinary.com/documentation/vue_integration)_
+## Table of Contents
+- [Key Features](#key-features)
+- [Version Support](#Version-Support)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Setup](#Setup)
+    - [Transform and Optimize Assets](#Transform-and-Optimize-Assets)
+    - [Generate Image and HTML Tags](#Generate-Image-and-Video-HTML-Tags)
 
-# 🛠️ Installation 
-## Vue 2.x
-1. Install using Vue-CLI
-    - After you create your application with Vue-CLI, navigate to the created app folder, and install Cloudinary SDK by:
-        ```bash
-        vue add cloudinary
-        ```
-    - Set up your cloudName and pick the components to use (or use all 😃)
-    ![Set up with cloudName and options](http://bit.ly/2WSKTf0)
-    - A `cloudinary.js` file will be added to your src directory, same level with your `main.js` file. Feel free to customize it if you like. And have fun with Cloudinary! 🤟
-    - More information on the plugin, check out [our Vue-CLI plugin for Cloudinary Repo](https://github.com/cloudinary/vue-cli-plugin-cloudinary)
+## Key Features
+- [Transform](https://cloudinary.com/documentation/vue_video_manipulation#video_transformation_examples) and [optimize](https://cloudinary.com/documentation/vue_image_manipulation#image_optimizations) assets (links to docs).
```

Comment:
Can we drop the "(link to docs)" at the end?

Reply:
We can, but I added it because there are many links in this read me,
I wanted it to be clear that it's an external link and not hijack people away from the readme.